### DISCUSSION TITLE
How to use configuration settings to specify the letter case of hexadecimal numbers.

### DISCUSSION THREAD

**Discussion by ahaoboy** [biomejs/biome#1554] [General]
https://biomejs.dev/playground/?code=YwBvAG4AcwB0ACAAdwBoAGkAdABlACAAPQAgADAAeABGAEYARgBGAEYARgA%3D

input:
```js
const white = 0xFFFFFF
```

output:
```js
const white = 0xffffff;
```

**Comment by Conaclos** [comment]
You can't. Biome follows Prettier conventions and normalizes to lowercase hexadecimal digits.
